docs/ota-client-guide/modules/ROOT/examples/start.sh

```diff
@@ -0,0 +1,366 @@
+  local pid=$!
+  trap "kill_pid ${pid}" EXIT
+  sleep 3s
+
+  local namespace="x-ats-namespace:default"
+  local api="http://localhost:${PROXY_PORT}/api/v1/namespaces/${NAMESPACE}/services"
+  local keyserver="${api}/tuf-keyserver/proxy"
+  local reposerver="${api}/tuf-reposerver/proxy"
+  local director="${api}/director/proxy"
+  local id
+  local keys
+
+  retry_command "director" "[[ true = \$(http --print=b GET ${director}/health \
+    | jq --exit-status '.status == \"OK\"') ]]"
+  retry_command "keyserver" "[[ true = \$(http --print=b GET ${keyserver}/health \
+    | jq --exit-status '.status == \"OK\"') ]]"
+  retry_command "reposerver" "[[ true = \$(http --print=b GET ${reposerver}/health/dependencies \
+    | jq --exit-status '.[].status == \"up\"') ]]"
+
+  id=$(http --ignore-stdin --check-status --print=b POST "${reposerver}/api/v1/user_repo" "${namespace}" | jq --raw-output .)
+  http --ignore-stdin --check-status POST "${director}/api/v1/admin/repo" "${namespace}"
+
+  retry_command "keys" "http --ignore-stdin --check-status GET ${keyserver}/api/v1/root/${id}"
+  keys=$(http --ignore-stdin --check-status GET "${keyserver}/api/v1/root/${id}/keys/targets/pairs")
+  echo ${keys} | jq '.[0] | {keytype, keyval: {public: .keyval.public}}'   > "${SERVER_DIR}/targets.pub"
+  echo ${keys} | jq '.[0] | {keytype, keyval: {private: .keyval.private}}' > "${SERVER_DIR}/targets.sec"
+
+  retry_command "root.json" "http --ignore-stdin --check-status -d GET \
+    ${reposerver}/api/v1/user_repo/root.json \"${namespace}\"" && \
+    http --ignore-stdin --check-status -d -o "${SERVER_DIR}/root.json" GET \
+    ${reposerver}/api/v1/user_repo/root.json "${namespace}"
+
+  echo "http://tuf-reposerver.${DNS_NAME}" > "${SERVER_DIR}/tufrepo.url"
+  echo "https://${SERVER_NAME}:30443" > "${SERVER_DIR}/autoprov.url"
+  cat > "${SERVER_DIR}/treehub.json" <<END
+{
+    "no_auth": true,
+    "ostree": {
+        "server": "http://treehub.${DNS_NAME}/api/v3/"
+    }
+}
+END
+
+  zip --quiet --junk-paths ${SERVER_DIR}/{credentials.zip,autoprov.url,server_ca.pem,tufrepo.url,targets.pub,targets.sec,treehub.json,root.json}
+
+  kill_pid "${pid}"
+  ${KUBECTL} create secret generic "user-keys" --from-literal="id=${id}" --from-literal="keys=${keys}"
+}
+
+
+[ $# -lt 1 ] && { echo "Usage: $0 <command> [<args>]"; exit 1; }
+command=$(echo "${1}" | sed 's/-/_/g')
+
+case "${command}" in
+  "start_all")
+    check_dependencies
+    start_weave
+    new_server
+    start_ingress
+    start_infra
+    start_vaults
+    start_services
+    ;;
+  "start_ingress")
+    start_ingress
+    ;;
+  "start_infra")
+    start_infra
+    ;;
+  "start_vaults")
+    start_vaults
+    ;;
+  "start_services")
+    start_services
+    ;;
+  "new_client")
+    new_client
+    ;;
+  "new_server")
+    new_server
+    ;;
+  "print_hosts")
+    print_hosts
+    ;;
+  "templates")
+    generate_templates
+    ;;
+  *)
+    echo "Unknown command: ${command}"
+    exit 1
+    ;;
+esac
```

Comment:
Do we have to have this file here? It has almost nothing to do with aktualizr; it's from OTA CE if memory serves. Can we instead reference it directly from that repo, or only copy the relevant parts?

Reply:
Well, we're waiting for an issue with our doc toolkit to be fixed (https://gitlab.com/antora/antora/issues/195). Then we can just fetch the file directly from the OTA CE repo. 
I used a copy of the file from the CE repo, because I was imaging in a workaround where we copy a fresh version of this file from the relevant repo at build time. We have a ticket for it here: https://saeljira.it.here.com/browse/OTA-3569
This file was a placeholder so the build would work in the meantime. I can hard-code it but I just need to remember to convert it back to an include when the fix/workaround is ready.

Reply:
I see. It's not a blocker, but actually the bigger issue is that we probably shouldn't be pulling that code from there. We have code in multiple places that does that same work, and I think it makes more sense to use a tool instead of a bunch of openssl commands.